regression-test/src/pages/_app.tsx

```diff
@@ -0,0 +1,20 @@
+import Head from 'next/head'
+import type { AppProps } from 'next/app'
+import { InstUISettingsProvider } from '@instructure/emotion'
+import { Alert, View } from '@instructure/ui'
```

Comment:
`Alert`, `View`, and the `* as Components` imports are unused, please delete them.